pkg/kncloudevents/good_client.go

```diff
@@ -1,26 +1,29 @@
 package kncloudevents
 
 import (
+	"github.com/cloudevents/sdk-go"
 	"github.com/cloudevents/sdk-go/pkg/cloudevents/client"
 	"github.com/cloudevents/sdk-go/pkg/cloudevents/transport/http"
```

Comment:
Delete these imports

Reply:
@n3wscott  `Option` is still in `http`,

see:  https://github.com/cloudevents/sdk-go/pull/116